Don't use f2c/clapack, as this depends on which version of lapack is installed
The native fortran interface is portable.

--- server/UtDynamicsSimulator/CMakeLists.txt~	2017-02-17 04:39:49.000000000 +0100
+++ server/UtDynamicsSimulator/CMakeLists.txt	2017-08-24 14:36:35.160932218 +0200
@@ -10,7 +10,7 @@
 
 add_executable(${program} ${sources})
 
-add_definitions(-DSEGA -DUSE_CLAPACK_INTERFACE)
+add_definitions(-DSEGA)
 include_directories(sDIMS)
 link_directories(sDIMS)
 
